app/controllers/users_controller.rb

```diff
@@ -0,0 +1,16 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+    if current_user?(@user)
+      @issues = @user.issues
+    else
+      redirect_back(fallback_location: root_path)
+    end
+  end
+
+  private
+
+  def current_user?(user)
```

Comment:
remove this method. no logic here and it's not needed + you already have "current_user" and now you making me confusing what this "?" mean. 
delete method